<!DOCTYPE html>
<html>
<body>
<h1>RibOxSwitch</h1>
<p> Synthetic biology promises the rational redesign of biological processes by controlling gene expression for applications throughout the pharmaceutical, brewing, and medical industries. This has largely been achieved through synthetic biology by repurposing natural mechanisms affecting translation, such as transcription factors and promoter sequences. However, these systems are not trivial to design and cannot be retargeted to arbitrary DNA sequences limiting their potential to be easily configured for new functions. </p>
<p>RNA switches have great potential for the design of synthetic genetic circuits. In comparison with protein-based transcriptional regulation, it may be more simple to design RNA switches through the relative simplicity of nucleotide interactions, which could have less stochasticity due to the larger pool of cytoplasmic RNA compared with smaller pool of transcription factors, and may have better kinetics due to the relatively fast turnover of RNA.</p>
<p>However, designs for synthetic riboswitches lack the inherent cooperativity of some protein-based regulators and natural riboswitches, leading to a broad dose-response rather than a ‘switch-like’ response to changes in concentration. <b>We aim to design cooperative riboswitches permitting rational control of gene expression.</b></p> 
<p>We take inspiration from the design of DNA aptamers and DNA molecular beacons, where it has been demonstrated that doubling the number of ‘receptor’ sites within a DNA switch increased cooperativity. By applying the same principle to an RNA hairpin within an mRNA transcript, we could selectively open and close the switch to reveal or hide a ribosome-binding site (RBS) and thus control protein translation. Importantly, the rate of translation from this RBS will be regulated cooperatively by the concentration of a ‘trigger’ RNA strand. This RNA strand, given that its sequence is not greatly constricted, could be an aptamer with specificity for the output of the system allowing the expression of a gene be related to the concentration of the product or another molecule.</p>
<p>In this project we intend to design and test a cooperative riboswitch, first demonstrating the unfolding of the RNA hairpin <i>in vitro</i> using PAGE and FRET. We then intend to show the same cooperative behaviour <i>in vivo</i> in a variety of constructs using <i>E. coli</i> cells and the expression of a fluorescent reporter as a model system. Future work would be to incorporate the cooperative riboswitch into more advanced genetic circuits such as positive and negative feedback systems. </p>
<img src="https://thesciencedog.files.wordpress.com/2013/09/golden-retriever-and-science1.jpg" width="50%"><figcaption>Lab work</figcaption>
<iframe width="420" height="315" src="https://www.youtube.com/watch?v=6A9M226_28Y"></iframe>
</body>
</html>
